curriculum/challenges/english/07-scientific-computing-with-python/learn-list-comprehension-by-building-a-case-converter-program/657ee28cefc4945568287673.md

```diff
@@ -7,9 +7,9 @@ dashedName: step-3
 
 # --description--
 
-Now that you have an empty list in place, you can start iterating through the input string and start converting each character to snake case.
+With the empty list in place, iterate through the input string and convert each character to snake case using a `for` loop. Iterating over a sequence is called traversal.
```

Comment:
we can't convert each character to snake case, as it's a thing of the whole string. Also written like that it looks like instructions and a camper would be extremely confused on how to convert each character to snake case using a for loop.

Also I don't think we need to add extra info like that it is called a traversal.

```suggestion
With the empty list in place, iterate through the input string to start converting the string to snake case character by character using a `for` loop.
```

Reply:
I've made adjustments to clarify that the snake case conversion applies to the entire string, not individual characters, and removed unnecessary information about traversal.